docs/contributing/contributing-rules.md

```diff
@@ -59,13 +59,30 @@ A well-written rule message includes:
 
 For an example of a good rule message, see: [this rule for Django's mark_safe](https://semgrep.dev/r?q=python.django.security.audit.avoid-mark-safe.avoid-mark-safe).
 
+Use the YAML multiline string operator `>-`. This operator enables you to write the rule message as one long line or multiple lines. Whichever way you choose, all will be interpreted as one line. This presents the best-looking rule message on the command-line.
+
+```yaml
+message: >-
```

Comment:
all this extra bit below I think is unnecessary.

Reply:
oh, I just saw @s-santillan thought this was good, so ... ignore me. It just felt like a long digression when I read it.

Reply:
Ah right... so you suggest deleting this? But I did not got where to put `>-` so I thought that an example might help...

Reply:
I mean, maybe we can do something more generic:

```yaml
message: >-
  This is a multiline
  comment example
  for Semgrep rules. 
```

I know it is boring... But we can delete the code snippet below showing the output, what do you think @efortuna?

Reply:
Ok, removing, looking at your comment above, this is too generic. Thank you!

Reply:
sweet! lgtm!